src/OpenLoco/Drawing/FPSCounter.cpp

```diff
@@ -0,0 +1,55 @@
+#include "FPSCounter.h"
+#include "../Graphics/Colour.h"
+#include "../Graphics/Gfx.h"
+#include "../Localisation/StringManager.h"
+#include "../Ui.h"
+
+#include <SDL2/SDL.h>
+#include <stdio.h>
+
+namespace OpenLoco::Drawing
+{
+    static uint32_t _lastFPSUpdateTicks;
+    static uint32_t _lastFPSTicks;
+    static float _currentFPS;
+
+    static float measureFPS()
+    {
+        const uint32_t currentTicks = SDL_GetTicks();
```

Comment:
I know its not absolutely necessary but in C++ we have ```std::chrono```, we don't have to rely on third party libraries anymore.